buildlib/pr/main.yml

```diff
@@ -151,6 +151,9 @@ resources:
     - container: ubuntu20_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu20.04-mofed5-cuda11:1
       options: $(DOCKER_OPT_VOLUMES) $(DOCKER_OPT_GPU)
+    - container: ubuntu20_rocm
+      image: registry.hub.docker.com/rocm/dev-ubuntu-20.04
```

Comment:
seems image is missing tag
A container resource with name ubuntu2004_rocm could not be found. The container resource does not exist. If you intended to specify an image, use NAME:TAG or NAME@DIGEST. For example, ubuntu:latest

Reply:
I appended :latest, lets see whether this works